### PR TITLE
Add ripple effect

### LINKED ISSUE #2343: Highlight items in the file browser on click
### Actual behaviour
When you tap on an item in the file browser, there is no indication that the tap was registered. Other apps usually turn the item background gray in this case. This is independent from the background change after a long tap.

This works correctly in the settings screen, if you click on an item there, the item background will turn gray while your finger is on the screen.

### Expected behaviour
Items in the file browser should be highlighted on touch, same as in the settings screen (and most other Android apps).
 
### Steps to reproduce
1. open file browser in nextcloud app
2. click an item
3. there is no onscreen indication that the click was successful
4. repeat the same steps on the settings screen, note the background change

### Environment data
Android version: 7.1

Device model: LG G4

Stock or customized system: LineageOS

Nextcloud app version: 3.0.3

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #2343 

In emulator it does not work for me when clicking a folder, but on a real device it works…

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
:+1:

**Comment by AndyScherzinger** [comment]
cc @mario @ardevd for second review/approval. I tested it on Android 8 on my OP3T device

**Comment by AndyScherzinger** [comment]
cc @mario @ardevd for second review/approval.

**Comment by AndyScherzinger** [comment]
cc @mario @ardevd for second review/approval. I tested it on Android 8 on my OP3T device

File: src/main/res/layout/grid_image.xml [hunk 1]
```diff
@@ -16,13 +16,14 @@
...
 -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/ListItemLayout"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_gravity="center_horizontal"
-    android:background="@drawable/list_selector"
-    android:gravity="center_horizontal"
-    android:orientation="vertical">
+              android:id="@+id/ListItemLayout"
```
Comment (ardevd) [inline src/main/res/layout/grid_image.xml]: Indentation issue.

Comment (ardevd) [reply src/main/res/layout/grid_image.xml]: @AndyScherzinger Hmm, ok. I guess it just looks odd when viewed on GitHub :) Though to be safe, you can simply do ctrl+a on the file and hit ctrl+alt+l to be sure :)

**Comment by AndyScherzinger** [comment]
fixed the formatting issue as mentioned by @ardevd

**Comment by ardevd** [comment]
:+1:

**Pull request merged**